Packs/MicrosoftECM/Integrations/MicrosoftECM/MicrosoftECM.yml

```diff
@@ -0,0 +1,781 @@
+      type: number
+    description: Gets Configuration Manager PowerShell scripts
+  - name: ms-ecm-script-create
+    arguments:
+    - name: script_file_entry_id
+      description: The Entry ID of the script file
+    - name: script_text
+      description: The text of the string
+    - name: script_name
+      required: true
+      description: The name of the script
+    outputs:
+    - contextPath: MicrosoftECM.Scripts.ApprovalState
+      description: The approval state of the script
+      type: string
+    - contextPath: MicrosoftECM.Scripts.Approver
+      description: The approver of the script
+      type: string
+    - contextPath: MicrosoftECM.Scripts.Author
+      description: The author of the script
+      type: string
+    - contextPath: MicrosoftECM.Scripts.Comment
+      description: A short comment about the script
+      type: string
+    - contextPath: MicrosoftECM.Scripts.LastUpdateTime
+      description: The date of the last script upda
+      type: date
+    - contextPath: MicrosoftECM.Scripts.Parameterlist
+      description: The parameter list of the script
+      type: string
+    - contextPath: MicrosoftECM.Scripts.Script
+      description: The code of the script
+      type: string
+    - contextPath: MicrosoftECM.Scripts.ScriptGuid
+      description: The unique identifier of the script
+      type: string
+    - contextPath: MicrosoftECM.Scripts.ScriptHash
+      description: The hash of the script
+      type: string
+    - contextPath: MicrosoftECM.Scripts.ScriptHashAlgorithm
+      description: The algorithm with which the script hash was generated
+      type: string
+    - contextPath: MicrosoftECM.Scripts.ScriptName
+      description: The name of the script
+      type: string
+    - contextPath: MicrosoftECM.Scripts.ScriptType
+      description: The type of the script
+      type: string
+    - contextPath: MicrosoftECM.Scripts.ScriptVersion
+      description: The version of the script
+      type: number
+    description: create new powershell script
+  - name: ms-ecm-script-invoke
+    arguments:
+    - name: script_guid
+      required: true
+      description: Specifies the script ID
+    - name: collection_id
+      description: Specifies the collection ID
+    - name: collection_name
+      description: Specifies the collection name
+    - name: device_name
+      description: Specifies a device name in Configuration Manager
+    outputs:
+    - contextPath: MicrosoftECM.ScriptsInvocation.OperationID
+      description: The unique Id of the script operation
+      type: number
+    - contextPath: MicrosoftECM.ScriptsInvocation.ReturnValue
+      description: The Return value of the script operation, 0 upon success
+      type: number
+    description: Invokes a script in Configuration Manager
+  - name: ms-ecm-script-approve
+    arguments:
+    - name: comment
+      required: true
+      description: Specifies a comment about the approval of the script
```

Comment:
mention how to retrieve

Reply:
This is a new comment that should be added with the script approval